docs/discover/kuery.asciidoc

```diff
@@ -0,0 +1,110 @@
+[[kuery-query]]
+=== Kuery
+
+experimental[This functionality is experimental and may be changed or removed completely in a future release.]
+
+Kuery is a new query language built specifically for Kibana. It aims to simplify the search experience in Kibana
+and enable the creation of helpful features like auto-complete, seamless migration of saved searches, additional
```

Comment:
@alexfrancoeur I already added a clarification per Lukas's suggestion, were you looking at the most recent changes?